rustls-mbedcrypto-provider/src/hmac.rs

```diff
@@ -93,3 +97,55 @@ impl MbedHmacContext {
         }
     }
 }
+
+/// A HMAC tag, stored as a value.
+#[derive(Clone)]
+pub(crate) struct Tag {
+    buf: [u8; Self::MAX_LEN],
```

Comment:
You could use `ArrayVec` instead (https://docs.rs/arrayvec/latest/arrayvec/). I don't have an objection to this either.

Reply:
I need a customized `Drop` implementation here, so let we keep using this type.